packages/headless/src/controllers/product-listing/category-facet/headless-product-listing-category-facet.ts

```diff
@@ -79,26 +79,30 @@ export function buildCategoryFacet(
 
     deselectAll: () => {
       coreController.deselectAll();
-      dispatch(fetchProductListing());
-      dispatch(logFacetClearAll(getFacetId()));
+      dispatch(fetchProductListing()).then(() =>
```

Comment:
Our `executeSearch` action takes an obligatory analytics argument, could this be a better pattern here?

Reply:
Yeah.... I semi-agree :P 

We were assuming from the start that we would log the product listings results from the back-end, so the `fetchProductListing` action doesn't really need to log its products from the front-end, which is why there was no analytics there.

But I have no idea how *components* interactions would be logged from this back-end perspective... How can the back-end figure out that the "next" button was clicked :/ 

So in the meantime, I just added those back here, and we'll figure things out eventually